frontend/admin/src/js/components/user/ViewUser.tsx

```diff
@@ -72,21 +72,15 @@ export const ViewUserPage: React.FC<ViewUserPageProps> = ({ user }) => {
           </h2>
         )}
         <div data-h2-margin="m(left, auto)">
-          <Link
-            mode="outline"
-            color="primary"
-            type="button"
-            href="/"
-            // download={ TODO }
-          >
+          <UserProfilePrintButton userId={user.id}>
             <span>
               <DownloadIcon style={{ width: "1rem" }} />{" "}
               {intl.formatMessage({
                 defaultMessage: "Download Profile",
```

Comment:
I wonder if this should be change to *Print Profile* with a printer icon?  It would make it a little clearer that a print dialog is about to appear, not a save file dialog.

Reply:
@Jerryescandon switching it to *Print Profile*?

Reply:
Switch it baby